src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -166,10 +169,33 @@ impl HummockManager {
             .await?;
 
         let modified_compaction_groups: Vec<_> = commit_sstables.keys().cloned().collect();
+        // fill compaction_groups
+        if let Some(compaction_group_manager) = compaction_group_manager_txn.as_ref() {
+            for cg_id in &modified_compaction_groups {
+                if !compaction_groups.contains_key(cg_id) {
+                    let compaction_group = compaction_group_manager
+                        .get(cg_id)
+                        .unwrap_or_else(|| panic!("compaction group {} should be created", cg_id))
+                        .compaction_config();
+                    compaction_groups.insert(*cg_id, (false, compaction_group));
```

Comment:
Will this overwrite what we do in L146? For example, will we mistakenly treat a new cg as a old cg by overwriting the bool to false?

Reply:
I don't think overwrite occurs, it is determined by compaction_groups.contains_key before insert.